activejob/lib/active_job/test_helper.rb

```diff
@@ -77,14 +77,23 @@ def queue_adapter_for_test
     #       HelloJob.perform_later('jeremy')
     #     end
     #   end
-    def assert_enqueued_jobs(number, only: nil)
+    #
+    # If a queue name is passed, the number of times jobs enqueued to that specific queue can be asserted.
+    #
+    #   def test_logging_job
+    #     assert_enqueued_jobs 1, queue: 'low' do
```

Comment:
This is pretty picky on my part, but I don't like the example of the queue name being "low" because it implies in this example that queue name is the priority. While often queue name does have relation to priority there is a priority setting so I'd rather use "default" like you did below.